src/lib-python/meteo_sounding_utils.py

```diff
@@ -0,0 +1,159 @@
+# --+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+
+# Set of routines to do interpolation/extrapolation of P/Z/T sounding data.
+# These routines should allow one to compute a full set of P/Z/T data given
+# the various sounding parts (e.g. TTAA (mandatory P,Z,T), TTBB (significant
+# P,T) and PPBB (winds given height alone).
+#
+# Peter Neilley, NCAR/RAP 8/93
+#
+# --+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+
+
+import math
+
+# Constants
```

Comment:
looks good @gthompsnJCSDA ,,, in the future think about how to abstract these

sure they are already existing in other parts of the code.